build.sh

```diff
@@ -218,6 +220,16 @@ test_ios_devices() {
     configuration="$3"
     failed=0
     for device in "${serial_numbers[@]}"; do
+        if [[ "$device" = "07510d12c09c53e44b3682dd0367934f82cfab9c" ]]; then
```

Comment:
Doesn't this result in the swift devices tests job just not running if the VM with the ios7 device gets picked? Removing it as on option using a tag for ios 8+ in jenkins seems better.